hedyorg/hedy
# 💻 Remove reload for deleting a class and an adventure

Fixes #5383 

**How to test**

1. Create a class and an adventure
2. Remove the class and adventure -> the page should not reload and the class/adventure should be removed right away.
3. Remove an adventure from inside an adventure

https://github.com/hedyorg/hedy/assets/48122190/cf87f738-71ef-40a2-9cb5-c798148282cc
